front/app/containers/Admin/projects/project/participationContext/components/ParticipationMethodPicker.tsx

```diff
@@ -64,6 +72,17 @@ export const ParticipationMethodPicker = ({
           />
         )}
       </SubSectionTitle>
+      {isExistingProjectOrPhase && (
+        <Box id="e2e-participation-method-warning" mb="24px">
+          <Warning>
+            {!isNilOrError(phase) ? (
```

Comment:
Minor. Since the element is the same for both, we could do 

```
<FormattedMessage {...(!isNilOrError(phase) ? messages.phaseMethodChangeWarning : messages.phaseMethodChangeWarning)} />
```